Workbooks/Azure Sentinel/Overview/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+    "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/settings.json",
+    "name":"Azure Sentinel Overview",
+    "author": "Microsoft",
+    "galleries": [{ "type": "sentinel", "resourceType": "Azure Monitor", "order": 100 }]
```

Comment:
and you can consider adding `{ "type": "workbook", "resourceType": "Azure Monitor", "categoryKey": "azureMonitorInsights", "order": 100 },` in this galleries array. This will make your insight show up on the main azure monitor gallery under the insights category when the feature flag `feature.includePreviewTemplates=true`
![image](https://user-images.githubusercontent.com/7664931/98585798-a9809f00-227c-11eb-8531-bdb22cdc1b6f.png)